sdk/core/Azure.Core/README.md

```diff
@@ -196,4 +201,20 @@ KeyVaultSecret secret = client.GetSecret("Name");
 
 More on mocking in [mocking samples](samples/Mocking.md)
 
+## Troubleshooting
+
+Three main ways of troubleshooting failures are [inspecting exceptions](samples/Response.md#Enabling exceptions), enabling [logging](samples/Diagnostics.md#Logging) and [distributed tracing](samples/Diagnostics.md#Distributed-tracing)
```

Comment:
Nit: I recommend an Oxford comma (Anne and I are pretty diligent in this, so it'd be consistent) after "enabling logging".